example/example_test.go

```diff
@@ -85,10 +85,22 @@ var cases = []struct {
 			"page_number": 1,
 			"result_per_page": 10]
 		}`,
-		ExpectedStatusCode:  422,
+		ExpectedStatusCode:  400,
 		ExpectedJSONResBody: ``,
 	},
 	{
+		Name: "test response nil",
+		URL:  "/example.SearchService/SearchReturnNil",
+		JSONReqBody: `{
+			"query": "query string",
+			"page_number": 1,
+			"result_per_page": 10
+		}`,
+		ExpectedStatusCode:  200,
+		ExpectedJSONResBody: ``,
```

Comment:
If we return a message, Then we need to define a proto in our prottp package and expose it to client, so that client could parse it and get the message?

```protobuf
message Error {
    string message = 1;
}
```